src/lib/components/CmsForm.svelte

```diff
@@ -26,14 +30,16 @@
 	let fullScreen = cmsPage?.fullScreen || false;
 	let maintenanceDisplay = cmsPage?.maintenanceDisplay || false;
 	let hideFromSEO = cmsPage?.hideFromSEO || false;
-
+	let hasCustomMeta = !!cmsPage?.metas?.length;
 	let hasMobileContent = cmsPage?.hasMobileContent || false;
 	let mobileContent = cmsPage?.mobileContent || '';
 	function confirmDelete(event: Event) {
 		if (!confirm('Would you like to delete this CMS page?')) {
 			event.preventDefault();
 		}
 	}
+	let metas = cmsPage?.metas || undefined;
```

Comment:
```suggestion
	let metas = cmsPage?.metas;
```

I don't understand what the ` || undefined` does